tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -42,6 +42,15 @@ auto make_testing_format_args(Args&&... vals) {
     }
 }
 
+template <class charT, class... Args>
+void throw_helper(const charT* fmt, const Args&... vals) {
+    try {
+        format(fmt, vals...);

```

Comment:
This should probably be perfect forwarding.

Reply:
`format` takes by const ref, so I think it's good as-is.